app/assets/javascripts/components/autocomplete.js

```diff
@@ -19,18 +19,28 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
   }
 
   Autocomplete.prototype.initAutoCompleteSelect = function ($select) {
+    var selectElement = $select
```

Comment:
Is there a reason why this is needed? Could we not just change the argument name?

Reply:
Updated.
The was no real reason, I just wanted the code to be as close to a copy as possible but it's probably not that important with more changes to the autocomplete likely to come.